ui/app/AppLayouts/Chat/views/CommunitySettingsView.qml

```diff
@@ -28,7 +30,8 @@ StatusSectionLayout {
     // TODO: get this model from backend?
     property var settingsMenuModel: root.rootStore.communityPermissionsEnabled ? [{name: qsTr("Overview"), icon: "show"},
                                                                                  {name: qsTr("Members"), icon: "group-chat"},
-                                                                                 {name: qsTr("Permissions"), icon: "objects"}] :
+                                                                                 {name: qsTr("Permissions"), icon: "objects"},
+                                                                                 /*{name: qsTr("Tokens"), icon: "token"}*/] :
```

Comment:
WDYT if you create a toggle in the `experimental features` section like we do with `Permissions` so we can also test it without modifying the code?